.github/workflows/macos-windows.yml

```diff
@@ -0,0 +1,64 @@
+name: macOS and Windows
+
+on:
+  push:
+    branches:
+      - main
+      - 1.4.x
+  pull_request:
+    branches:
+      - main
+      - 1.4.x
+    paths-ignore:
+      - "doc/**"
+
+env:
+  PANDAS_CI: 1
+  PYTEST_TARGET: pandas
+  PYTEST_WORKERS: auto
+  PATTERN: "not slow and not db and not network and not single_cpu"
+
+jobs:
+  pytest:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash -el {0}
+    timeout-minutes: 120
+    strategy:
+      matrix:
+        # TODO
+        # os: [macos-10.15, windows-2019]
+        os: [windows-2019]
+        env_file: [actions-38.yaml, actions-39.yaml, actions-310.yaml]
+        pyarrow_version: [6]
+      fail-fast: false
+    name: ${{ matrix.name || format('{0} {1} pyarrow={2}', matrix.os, matrix.env_file, matrix.pyarrow_version) }}
+    concurrency:
+      # https://github.community/t/concurrecy-not-work-for-push/183068/7
+      group: ${{ github.event_name == 'push' && github.run_number || github.ref }}-${{ matrix.os }}-${{ matrix.env_file }}-${{ matrix.pyarrow_version }}-mac-win
+      cancel-in-progress: true
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: wmic.exe
```

Comment:
Can you add a name that helps understand what this is about please

Reply:
Idk what it does, seems to print some CPU info, it's copied from Azure CI scripts